tests/utils-e2e.ts

```diff
@@ -19,11 +18,16 @@ import {
   Address,
   Token,
   TransferFeeParams,
+  Config,
 } from '../src/types';
 import Erc20ABI from '../src/abi/erc20.json';
 import AugustusABI from '../src/abi/augustus.json';
 import { generateConfig } from '../src/config';
 import { DummyLimitOrderProvider } from '../src/dex-helper';
+import { constructSimpleSDK, SimpleFetchSDK, SimpleSDK } from '@paraswap/sdk';
+import { sleep } from '../tests/utils';
+import axios from 'axios';
+import { SmartToken, StateOverrides } from './smart-tokens';
```

Comment:
I just wonder, if we really need this `SmartToken` thing. It will require a lot of change. What if we just change `holders` -> to variable names need to override. It should be enough. What do you think, guys? @Louis-Amas @ColonelJ @mwamedacen

Reply:
Btw what you propose is what I started to build, but I found out it's better to create smart token class because it permit to hide also approval and balance for srcToken.